thrall/app/lib/ElasticSearch.scala

```diff
@@ -131,19 +131,25 @@ object ElasticSearch extends ElasticSearchClient with ImageFields {
       .executeAndLog(s"removing exports from image $id")
       .incrementOnFailure(failedExportsUpdates) { case e: VersionConflictEngineException => true }
 
-  def applyImageMetadataOverride(id: String, metadata: JsValue)(implicit ex: ExecutionContext): Future[UpdateResponse] =
+  def applyImageMetadataOverride(id: String, metadata: JsValue)(implicit ex: ExecutionContext): Future[UpdateResponse] = {
+    val userMetaData = metadata \ "data"
+    val lastModified = metadata \ "lastModified"
     prepareImageUpdate(id)
       .setScriptParams(Map(
-        "userMetadata" -> asGroovy(metadata),
-        "lastModified" -> asGroovy(JsString(currentIsoDateString))
+        "userMetadata" -> asGroovy(userMetaData),
+        "lastModified" -> asGroovy(lastModified)
       ).asJava)
       .setScript(
-        "ctx._source.userMetadata = userMetadata;" +
-          refreshEditsScript +
-          updateLastModifiedScript,
+      s"""| if (ctx._source.lastModified && ctx._source.lastModified > lastModified) {} else {
```

Comment:
Isn't it using the `lastModified` set here: https://github.com/guardian/grid/pull/1599/files#diff-2454b3b50883ce82c6498f0096e91dc1R198 ?